webview_panels/src/uiCore/components/gradientButton/GradientButton.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button as ReactStrapButton, ButtonProps } from "reactstrap";
+import Tooltip from "../tooltip/Tooltip";
+import classes from "./styles.module.scss";
+
+export const GradientButton = (props: ButtonProps): JSX.Element => (
```

Comment:
add a sample in webview_panels\src\uiCore\uiToolkitStories\Components.stories.tsx